tasks/main.yml

```diff
@@ -130,7 +130,7 @@
 
 - name: Copy JBoss EAP
   copy:
-    src: "{{ jboss_eap_artifact_name }}"
+    src: "{{ jboss_eap_artifact_url }}"
```

Comment:
This modification causes issues when attempting to use the `transfer_method: copy-from-controller`